packages/clay-color-picker/src/Hue.tsx

```diff
@@ -20,66 +20,56 @@ type Props = {
 	value: number;
 };
 
-const useIsomorphicLayoutEffect =
-	typeof window === 'undefined' ? React.useEffect : React.useLayoutEffect;
-
 /**
  * Renders Hue component
  */
 const ClayColorPickerHue = ({value = 0, onChange = () => {}}: Props) => {
 	const containerRef = React.useRef<HTMLDivElement>(null);
-	const selectorActive = React.useRef<boolean>(false);
 
-	const {onPointerMove, setXY, x, y} = usePointerPosition(containerRef);
+	const {setXY, y} = usePointerPosition(containerRef);
 
-	const removeListeners = () => {
-		selectorActive.current = false;
+	const [internalValue, setInternalValue] = useState<number>(value);
 
-		window.removeEventListener('pointermove', onPointerMove);
-		window.removeEventListener('pointerup', removeListeners);
-	};
+	const handleOnChangeEnd = (event: any) => {
+		onChange(internalValue);
+
+		setXY({x: internalValue, y});
 
-	useIsomorphicLayoutEffect(() => {
-		if (containerRef.current && selectorActive.current) {
-			onChange(xToHue(x, containerRef.current));
-		}
-	}, [x]);
+		event.target.blur();
+
+		event.target.focus();
+	};
 
 	React.useEffect(() => {
-		if (containerRef.current) {
-			setXY({x: hueToX(value, containerRef.current), y});
-		}
+		setInternalValue(value);
```

Comment:
As we are going to use the parent state `value` and `onChange` we no longer need this to synchronize the values.

Reply:
I couldn't do this one due to https://github.com/liferay/clay/pull/5156#issuecomment-1295093670.